iothub/device/src/Transport/Mqtt/MqttIotHubAdapterFactory.cs

```diff
@@ -23,8 +23,8 @@ public MqttIotHubAdapterFactory(IotHubClientMqttSettings settings)
             return new MqttIotHubAdapter(
                 connectionCredentials.DeviceId,
                 connectionCredentials.ModuleId,
-                connectionCredentials.GatewayHostName,
-                connectionCredentials.Certificate == null ? connectionCredentials : null,
+                connectionCredentials.HostName,
+                connectionCredentials,
```

Comment:
If we pass in the connecitonCredentials object, no reason to pass the previous 3 as the class can get those values directly now.

Reply:
good catch, sure